Should be fine to use `self._id` there.

Test failed:
```
FAIL: test_process_change_with_deleted_document (corehq.blobs.tests.test_pillow.TestBlobDeletionProcessor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/blobs/tests/test_pillow.py", line 21, in test_process_change_with_deleted_document
    self.obj.fetch_attachment("name")
  File "/usr/local/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "/vendor/lib/python2.7/site-packages/testil.py", line 244, in raises
    eq_(str(err), msg)
  File "/vendor/lib/python2.7/site-packages/testil.py", line 54, in eq
    raise AssertionError(err)
AssertionError: not equal
"FakeCouchDocument 96e019833dc64d4b8ec52333245f5eb3 attachment: 'name'"
"FakeCouchDocument attachment: 'name'"
```

--- a/corehq/blobs/tests/test_pillow.py
+++ b/corehq/blobs/tests/test_pillow.py
@@ -16,7 +16,7 @@ class TestBlobDeletionProcessor(BaseTestCase):
         self.obj.put_attachment("content", "name")
         change = Config(id=self.obj._id, deleted=True)
         self.processor.process_change(None, change)
-        msg = "FakeCouchDocument attachment: 'name'"
+        msg = "FakeCouchDocument {} attachment: 'name'".format(self.obj._id)
         with assert_raises(ResourceNotFound, msg=msg):
             self.obj.fetch_attachment("name")
 

